### PR TITLE
added timeouts and TTLs to all HTTP clients

### PR DESCRIPTION
**Pull request by alexkorotkikh**
Issue #103

### CODE REVIEW

File: src/main/java/de/zalando/zmon/dataservice/data/HttpEventLogger.java [hunk 1]
```diff
@@ -69,7 +69,12 @@ public HttpEventLogger(DataServiceMetrics metrics, DataServiceConfigProperties c
         if (enabled) {
             forwardUrl = config.getEventlogUrl() + "/api/v1";
             log.info("EventLog enabled: {}", forwardUrl);
-            executor = Executor.newInstance(ProxyWriter.getHttpClient(config.getEventlogSocketTimeout(), config.getEventlogTimeout(), config.getEventlogConnections()));
```
Comment (mohabusama) [inline src/main/java/de/zalando/zmon/dataservice/data/HttpEventLogger.java]: Do we still need the ProxyWriter?

Comment (mohabusama) [reply src/main/java/de/zalando/zmon/dataservice/data/HttpEventLogger.java]: ah, right, only the client initialization is removed.

**Comment by mohabusama** [comment]
👍

**Comment by alexkorotkikh** [comment]
:+1:

**Pull request merged**